Tests/Unit/MSFT_xIISFeatureDelegation.tests.ps1

```diff
@@ -25,12 +25,7 @@ if (($repoSource -ne $null) -and ($repoSource[0].RepositorySourceLocation.Host -
         }
         
     }
-    Push-Location
-    cd (Join-Path -Path $moduleRoot -ChildPath '\Tests\DscResourceTestHelper')
-    $verFolder = Get-ChildItem
-    Pop-Location
-    $testModuleVer = $verFolder.Name
-    Write-Host "testModule = $verFolder, version = $testModuleVer"
+    $testModuleVer = Get-ChildItem -Path (Join-Path -Path $moduleRoot -ChildPath '\Tests\DscResourceTestHelper')
```

Comment:
I don't see removal of the prompt here as states in the description, but that's good actually.
Instead of removing it completely, can you check for environment variable $env:getDscTestHelper to be $true instead of relying on user inputting 'y'?